misc/idp-sync/api.md

```diff
@@ -0,0 +1,335 @@
+# Integration API Documentation
+
+## Introduction
+This reference provides detailed information on managing integrations via NetBird Cloud API.
+
+## Authentication
+Authentication is required for all API requests. Please refer to the [authentication guideline](https://docs.netbird.io/how-to/access-netbird-public-api) for how to create and authenticate API calls using Personal Access Tokens (PAT).
+
+## Google Endpoints
```

Comment:
I would call it Google Workspace or something like that. Google Endpoints sounds too vague.